doc/notebooks/GSoC2021_GRASS_Jupyter_Integration.ipynb

```diff
@@ -0,0 +1,90 @@
+{
+ "cells": [
+  {
+   "source": [
+    "# Improved Integration of GRASS and Jupyter\n",
+    "\n",
+    "As part of Google Summer of Code, 2021, we've been working to shorten and simplify the launch of GRASS in Jupyter and improve the map displays. You can find out more about the project and follow the progress on the [GRASS wiki page](https://trac.osgeo.org/grass/wiki/GSoC/2021/JupyterAndGRASS).\n",
+    "\n",
+    "This notebook is designed to run in binder and demonstrate the usage of `grass.jupyter`, the new module of Jupyter-specific functions for GRASS. "
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import subprocess\n",
+    "import sys"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gisbase = subprocess.check_output([\"grass\", \"--config\", \"path\"], text=True).strip()\n",
+    "os.environ['GISBASE'] = gisbase\n",
+    "sys.path.append(os.path.join(gisbase, \"etc\",\"python\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import grass.script as gs\n",
+    "import grass.jupyter as gj"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Let's check our import by printing the __doc__ for init()\n",
+    "print(gj.init.__doc__)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Start GRASS Session\n",
+    "gj.init(\"$HOME/Documents/grassdata\", \"nc_basic_spm_grass7\", \"user1\")\n",
+    "\n",
+    "# Set Display Settings\n",
+    "gj.display_settings(font=\"romans\") # Default font is \"sans\", let's change it to Romans just for fun!"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "name": "python383jvsc74a57bd07e402764773cbeaa2d2d6f3815f29bb5da9599569c66a0a6d6a02c360a55847f",
```

Comment:
I have seen issues before with conda specifying its complex kernel name and then the notebook failing elsewhere because the very specific kernel is not available. I'm not sure if it is the case here or not, but we have to check this time to time and will need to modify these by hand if needed. Jupyter Notebooks are unfortunately not as great with Git as one would wish.